src/js/view/view.js

```diff
@@ -711,6 +710,10 @@ define([
                 case states.PLAYING:
                     _resizeMedia();
                     break;
+                case states.PAUSED:
+                    if (_model.get('controls')) {
+                        _captionsRenderer.resize();
+                    }
```

Comment:
You only need to call `_captionsRenderer.renderCues(true);` This does less work and is what is used in other places to reposition text above the cb.